presto-raptor/src/main/java/com/facebook/presto/raptor/storage/FileStorageService.java

```diff
@@ -140,6 +147,22 @@ public static File getFileSystemPath(File base, UUID shardUuid)
                 .toFile();
     }
 
+    private void deleteFilesAsync(List<File> files)
+    {
+        if (files != null && !files.isEmpty()) {
+            executor.submit(() -> {
+                for (File file : files) {
+                    try {
+                        Files.deleteIfExists(file.toPath());
+                    }
+                    catch (IOException e) {
+                        log.warn("Failed to delete file: %s", file.getAbsolutePath());
```

Comment:
Log the original exception -- failure to delete a local file is very unexpected, so knowing what happened is useful